src/ServicePulse.Host/vue/src/composables/serviceRedirects.js

```diff
@@ -90,7 +90,7 @@ export async function useCreateRedirects(sourceEndpoint, targetEndpoint) {
     return result;
   } catch (err) {
     console.log(err);
-    var result_1 = {
+    const result_1 = {
```

Comment:
just return

Reply:
I think there is an eslint about this, "unnecessary variable", can you check @PhilBastian ?

Reply:
![Uploading image.png…]()